okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -1139,4 +1139,33 @@ public void redirectUsingNewConnectionEventSequence() throws IOException {
         "CallEnd");
     assertEquals(expectedEvents, listener.recordedEventTypes());
   }
+
+  @Test public void applicationInterceptorProceedsMultipleTimes() throws Exception {
```

Comment:
Any change to the currently ignored tests?

Reply:
Will do a pass when things settle down.